app/Http/Controllers/CourseController.php

```diff
@@ -14,6 +14,9 @@ class CourseController extends Controller
 {
     public function index(Request $request)
     {
+        if (auth()->user() && auth()->user()->role === 'admin') {
+            return redirect('/admin/');
+        }
```

Comment:
why we redirect admin from here?

Reply:
I suppose, there's no need to show the dashboard for admin.